test/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:12
+
+ENV password=notpassword1
+ENV dbpass=notpassword2
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- 56360491-584c-4ab1-8197-c84f87adf185 -->